success-stories/index.html

```diff
@@ -0,0 +1,50 @@
+---
+layout: success-stories 
+title:  "Success Stories"
+intro: "Real world use cases from Global <strong>2000 companies</strong> about the impact of using <strong>Spinnaker</strong> in their organizations."
```

Comment:
Avast isn't a G2K and neither are Lookout or Makerbot. Can we replace it with something else, e.g. "leading enterprises"